consoles/VNC.pm

```diff
@@ -135,7 +134,9 @@ sub login ($self, $connect_timeout = undef) {
     my $hostname = $self->hostname || 'localhost';
     my $port = $self->port || 5900;
     my $description = $self->description || 'VNC server';
-
+    my $local_timeout = $bmwqemu::vars{VNC_CONNECT_TIMEOUT_LOCAL} // 10;
+    my $remote_timeout = $bmwqemu::vars{VNC_CONNECT_TIMEOUT_REMOTE} // 60;
+    $connect_timeout //= ($hostname =~ qr/(localhost|127\.0\.0\.\d+|::1)/) ? $local_timeout : $remote_timeout;
```

Comment:
@kraih Does Perl or some module like Mojolicious provide a helper for this distinction? (e.g. something in the lines of https://github.com/Martchus/syncthingtray/blob/8e6bd62030db1867fd68d3798a7c1e497933bae5/connector/utils.cpp#L118)

Reply:
This should probably be a utility function with tests.